packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/drop/useDrop.tsx

```diff
@@ -102,8 +89,11 @@ export const useDrop = (): UseDropValue => {
       if (data === '') {
         return;
       }
+      const dropPosition = reactflow.screenToFlowPosition({
```

Comment:
Nodes are not dropped where I am expecting it at all. Please fix this part of restore the old code.

Reply:
Yes I had changes in my stash, the dependency of the useCallback should have been `reactflow`. I took the example from [there](https://reactflow.dev/examples/interaction/drag-and-drop)
I pushed an update.